odoo/addons/base/models/assetsbundle.py

```diff
@@ -99,6 +99,12 @@ def subber(match):
     ).strip()
     return result
 
+try:
+    import rjsmin as rjsminlib
+    rjsmin = rjsminlib.jsmin
+except ImportError:
```

Comment:
If we don't remove the regex, I would at least add a warning that the lib is missing.